docs/_plugins/shortcodes/repoStatus.cjs

```diff
@@ -1,55 +1,330 @@
  * @this {EleventyContext}
  */
-function repoStatus({ heading = 'Repo status', type = 'Pattern', level = 2 } = {}) {
-  return '';
-  // https://github.com/RedHat-UX/red-hat-design-system/issues/1174
-  // eslint-disable-next-line no-unreachable
-  const headingLevel = Array.from({ length: level }, () => '#').join('');
-  const checkSVG = fs.readFileSync('node_modules/@patternfly/icons/fas/check.svg', 'utf8');
-  /** @type {string[][]} */
+function getRepoData() {
   const docsPage = this.ctx._;
   const allStatuses = this.ctx.repoStatus ?? docsPage?.repoStatus ?? [];
   const title = this.ctx.title ?? docsPage?.title;
-  const [header, ...repoStatus] = allStatuses;
-  if (Array.isArray(header)) {
-    header[1] = type;
+  return allStatuses.find(component => component.name === title && component.type === 'Element').libraries;
+}
+
+/**
+ * Calls getRepoData function and outputs a definition list for each component
+ * @this {EleventyContext}
+ */
+function repoStatusList({ heading = 'Status', level = 2 } = {}) {
+  // Removing Documentation status from the repoStatusList
+  const statusList = getRepoData.call(this).filter(repo => repo.name !== 'Documentation');
+
+  if (!Array.isArray(statusList) || !statusList.length) {
+    return '';
+  } else {
+    return /* html */`
+<section class="section section--palette-default container">
+  <div class="component-status-list-heading">
+    <copy-permalink class="h${level}">
+      <h${level} id=${heading.toLowerCase()} tabindex="-1">
+        <a class="heading-anchor" href="#${heading.toLowerCase()}">${heading}</a>
+      </h${level}>
+    </copy-permalink>
+    <p><a href="#status-checklist">What do these mean?</a></p>
+  </div>
+
+  <div class="component-status-list-container">
+    <dl>
+        ${statusList.map(listItem => {
+    return /* html */`
+  <div>
+<dt>${listItem.name}</dt>
+<dd>
+<rh-tag color=${STATUS_LEGEND[listItem.status].color} variant=${STATUS_LEGEND[listItem.status].variant}>
+${listItem.status}${STATUS_LEGEND[listItem.status].icon}
+</rh-tag>
+</dd>
+</div>`;
+  }).join('\n').trim()}
+    </dl>
+  </div>
+</section>`;
   }
-  const bodyRows = repoStatus.filter(([id]) => id === title);
-  if (!Array.isArray(bodyRows) || !bodyRows.length) {
+}
+
+/**
+ * Reads component status data from global data (see above) and outputs a table for Design/Code status page
+ * @this {EleventyContext}
+ */
+function repoStatusTable() {
+  const docsPage = this.ctx._;
+  const allStatuses = this.ctx.repoStatus ?? docsPage?.repoStatus ?? [];
+  // Filtering out 'Responsive' status from all the libraries
+  const elementsList = allStatuses.map(item => ({ ...item, libraries: item.libraries.filter(lib => lib.name !== 'Responsive') }));
+
+  if (!Array.isArray(elementsList) || !elementsList.length) {
     return '';
   } else {
-    return /* html*/`
+    return /* html */`
+<div class="component-status-table">
+<rh-table>
```

Comment:
@bennypowers this is the checklist table to be used on all the "Overview" tab in docs.